src/Core/CoreKernel.php

```diff
@@ -498,10 +506,15 @@ protected function bootManifests($flush)
                 $config->setFlush(true);
             }
         }
+        // tell modules to sort, now that config is available
+        $this->getModuleLoader()->getManifest()->sort();
```

Comment:
Good point. We may need to update `ModuleManifest::activateConfig()` to work on a reversed array.